cmake/dependencies.cmake

```diff
@@ -122,12 +124,37 @@ function(ov_download_tbb)
                 ENVIRONMENT "TBBROOT"
                 SHA256 "321261ff2eda6d4568a473cb883262bce77a93dac599f7bd65d2918bdee4d75b")
     elseif(APPLE AND X86_64)
+        # build oneTBB 2021.2.1 with OS version 11.4
         RESOLVE_DEPENDENCY(TBB
                 ARCHIVE_MAC "oneapi-tbb-2021.2.1-mac.tgz"
                 TARGET_PATH "${TEMP}/tbb"
                 ENVIRONMENT "TBBROOT"
                 SHA256 "c57ce4b97116cd3093c33e6dcc147fb1bbb9678d0ee6c61a506b2bfe773232cb"
                 USE_NEW_LOCATION TRUE)
+    elseif(WIN32 AND AARCH64)
+        # build oneTBB 2021.2.1 with Visual Studio 2022 (MSVC 14.35)
```

Comment:
which toolset was used? v143?
Let's specify this information in future

Reply:
@ilya-lavrenov What do you mean "toolset"? Compiler + IDE combination?

Reply:
https://cmake.org/cmake/help/latest/variable/MSVC_TOOLSET_VERSION.html